vmm/src/vm.rs

```diff
@@ -2116,6 +2116,12 @@ impl Vm {
             self.vm.tdx_finalize().map_err(Error::FinalizeTdx)?;
         }
 
+        self.memory_manager
+            .lock()
+            .unwrap()
+            .create_userspace_mappings()
```

Comment:
Please add a large comment to explain the reason for this new function to be invoked here.

Reply:
Ack I will do that in the upcoming revision.